tensorflow/tensorboard/components/vz-line-chart/vz-line-chart.ts

```diff
@@ -71,7 +71,7 @@ module VZ {
       this.xScale = xComponents.scale;
       this.xAxis = xComponents.axis;
       this.xAxis.margin(0).tickLabelPadding(3);
-      this.yScale = new Plottable.Scales.Linear();
+      this.yScale = yScale === 'log' ? new Plottable.Scales.ModifiedLog() : new Plottable.Scales.Linear();
```

Comment:
Since "log" and "linear" are the accepted options, I think it would be better to do:

function getYScaleFromType(yScaleType: string): Plottable.QuantitativeScale {
  if (yScaleType === "log") {
    return new Plottable.Scales.ModifiedLog();
  } else if (yScaleType === "linear") {
    return new Plottable.Scales.Linear();
  } else {
    throw new Error("Unrecognized yScale type " + yScaleType);
  }